srcpkgs/xdg-desktop-portal-wlr/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'xdg-desktop-portal-wlr'
+pkgname=xdg-desktop-portal-wlr
+version=0.1.0
+revision=1
+build_style=meson
+configure_args="-Dsystemd=disabled"
```

Comment:
The absence of `systemd` is autodetected in `meson.build`, so this line is redundant IMHO.